local-cli/runIOS/runIOS.js

```diff
@@ -27,15 +27,24 @@ function runIOS(argv, config) {
 }
 
 function _runIOS(argv, config, resolve, reject) {
-  const args = parseCommandLine([{
-    command: 'simulator',
-    description: 'Explicitly set simulator to use',
-    type: 'string',
-    required: false,
-    default: 'iPhone 6',
-  }], argv);
+  const args = parseCommandLine([
+    {
+      command: 'simulator',
+      description: 'Explicitly set simulator to use',
+      type: 'string',
+      required: false,
+      default: 'iPhone 6',
+    },
+    {
+      command: 'ios-project-path',
```

Comment:
I'd suggest to discuss the change for android in another PR. There is already a parameter "root", which seems to do the same job. Maybe we just rename it or add an alias (for backwards compability). Also we have no working sample app for android which is runnable via run-android (needs a android/gradlew file?).